pkg/satellite/validation.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"regexp"
+	"strings"
+)
+
+// IsValid is a method for validating UserInfo entity
+func (userInfo *UserInfo) IsValid() bool {
+	if userInfo.FirstName == "" {
+		return false
+	}
+	if userInfo.LastName == "" {
+		return false
+	}
+	if !validateEmail(userInfo.Email) {
+		return false
+	}
+	if len(userInfo.Password) < 6 {
+		return false
+	}
+
+	return true
+}
+
+// validateEmail is a function for email validation
+func validateEmail(email string) bool {
```

Comment:
https://golang.org/pkg/net/mail/#ParseAddress

Reply:
Also the only correct way to validate an email is to actually send an email and let person click the confirmation link.

Reply:
The correct name for this would be `checkEmailAddressSyntax` or something like it.

Reply:
@egonelbre does it make sense to replace this function with some simple regexp? Just to simplify everything

Reply:
No, use a proper parser https://golang.org/pkg/net/mail/#ParseAddress. 

Email parsing is pretty difficult with a regex. https://stackoverflow.com/a/201378/192220, https://www.regular-expressions.info/email.html